src/date.ts

```diff
@@ -85,18 +76,18 @@ export class _Date {
    * @param to
    * @param num
    */
-  betweens(from: string, to: string, num?: number): Date[] {
+  betweens(from: string | Date, to: string | Date, num?: number): Date[] {
     if (typeof num == 'undefined') {
       num = 3;
     }
     const newDates: Date[] = [];
-    let fromMilli = Date.parse(from);
-    const dateOffset = (Date.parse(to) - fromMilli) / (num + 1);
-    let lastDate: string | Date = from;
+    const toMilli = to instanceof Date ? to.getTime() : Date.parse(to);
+    let fromMilli = from instanceof Date ? from.getTime() : Date.parse(from);
```

Comment:
You could extract these also to a private helper function

```suggestion
    const toMilli = this.toMillis(to);
    let fromMilli = this.toMillis(from);
```

It could make sense to rename `createDate` to `toDate`

Reply:
Done x2